gdcdictionary/schemas/clinical.yaml

```diff
@@ -39,59 +38,48 @@ required:
 uniqueKeys:
   - [id]
 
-# clinical properties
 properties:
-  $ref: "_definitions.yaml#/ubiquitous_properties"
+
+  $ref:
+    - "_definitions.yaml#/ubiquitous_properties"
 
   age_at_diagnosis:
+    description: 'Age in days at which a condition or disease was first diagnosed.
+      CDE: 3225640. TCGA has the age in years with CDE 2006657, this is converted
+      by ceil(age*365.25) to get days.'
     type: number
-    description: >
-      Age in days at which a condition or disease was first diagnosed.
-      CDE: 3225640. TCGA has the age in years with CDE 2006657,
-      this is converted by ceil(age*365.25) to get days.
 
   days_to_death:
+    description: "Time interval from a person's date of death to the date of initial\
+      \ pathologic diagnosis, represented as a calculated number of days. CDE: 3165475."
```

Comment:
This is an example where double-quotes are introduced.  It removes internal line breaks with escaping but keeps the line break at the end.